pickle-processor/src/main/java/com/fourlastor/pickle/ScenarioOutlineConverter.kt

```diff
@@ -0,0 +1,39 @@
+package com.fourlastor.pickle
+
+import gherkin.ast.Examples
+import gherkin.ast.Scenario
+import gherkin.ast.ScenarioOutline
+import gherkin.ast.Step
+import gherkin.ast.TableCell
+import gherkin.ast.TableRow
+
+internal fun Examples.toScenarios(outline: ScenarioOutline): List<Scenario> {
```

Comment:
New version of Cucumber is doing this Example to Scenario conversion during runtime. 

I copied from old version. Thanks to Kotlin, it is pretty straight-forward.